pkg/fanal/analyzer/os/ubuntu/esm.go

```diff
@@ -0,0 +1,93 @@
+package ubuntu
+
+import (
+	"bufio"
+	"context"
+	"os"
+
+	"encoding/json"
+
+	"golang.org/x/exp/slices"
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/trivy/pkg/fanal/analyzer"
+	aos "github.com/aquasecurity/trivy/pkg/fanal/analyzer/os"
+	"github.com/aquasecurity/trivy/pkg/fanal/types"
+)
+
+func init() {
+	analyzer.RegisterAnalyzer(&ubuntuESMAnalyzer{})
+}
+
+const (
+	ESMAnalyzerVersion = 1
+	esmConfFilePath    = "var/lib/ubuntu-advantage/status.json"
+	esmServiceName     = "esm-infra"
+	esmStatusEnabled   = "enabled"
+)
+
+var ESMRequiredFiles = []string{
+	esmConfFilePath,
+}
+
+type ubuntuESMAnalyzer struct{}
+
+func (a ubuntuESMAnalyzer) Analyze(_ context.Context, input analyzer.AnalysisInput) (*analyzer.AnalysisResult, error) {
+	scanner := bufio.NewScanner(input.Content)
```

Comment:
Is this file [JSON Lines](https://jsonlines.org/examples/)? Doesn't `json.NewDecoder(input.Content).Decode` work?

Reply:
yes, you are right. It works. I changed this.